bothub/api/v2/repository/views.py

```diff
@@ -715,21 +719,55 @@ def evaluate(self, request, **kwargs):
         user_authorization = repository.get_user_authorization(request.user)
         if not user_authorization.can_write:
             raise PermissionDenied()
-        serializer = EvaluateSerializer(data=request.data)  # pragma: no cover
-        serializer.is_valid(raise_exception=True)  # pragma: no cover
-
-        try:
-            request = repository.request_nlp_manual_evaluate(  # pragma: no cover
-                user_authorization, serializer.data
-            )
-        except DjangoValidationError as e:
-            raise APIException(e.message, code=400)
+        data = request.data
+        response = []
+        version_languages = RepositoryVersionLanguage.objects.filter(repository_version__pk=data.get("repository_version"))
```

Comment:
this line is long exceeding 119 characters, you can adjust it like this:

`        version_languages = RepositoryVersionLanguage.objects.filter(
            repository_version__pk=data.get("repository_version")
        )`